frontend/src/pages/Landing/index.js

```diff
@@ -191,17 +242,100 @@ function Landing() {
             <SimpleBar>
               <Container className="landing inline-size" padding={0}>
                 <Table className="usa-table usa-table--borderless usa-table--striped">
-                  <caption>Activity reports</caption>
+                  <caption>
+                    Activity reports
+                    <span className="smart-hub--table-nav">
+                      <span className="smart-hub--total-count">
+                        {renderTotal(offset, perPage, activePage, reportsCount)}
+                        <Pagination
+                          hideFirstLastPages
+                          prevPageText="<Prev"
+                          nextPageText="Next>"
+                          activePage={activePage}
+                          itemsCountPerPage={perPage}
+                          totalItemsCount={reportsCount}
+                          pageRangeDisplayed={4}
+                          onChange={handlePageChange}
+                          linkClassPrev="smart-hub--link-prev"
+                          linkClassNext="smart-hub--link-next"
+                        />
+                      </span>
+                    </span>
+                  </caption>
                   <thead>
                     <tr>
-                      <th scope="col">Report ID</th>
-                      <th scope="col">Grantee</th>
-                      <th scope="col">Start date</th>
-                      <th scope="col">Creator</th>
-                      <th scope="col">Topic(s)</th>
-                      <th scope="col">Collaborator(s)</th>
-                      <th scope="col">Last saved</th>
-                      <th scope="col">Status</th>
+                      <th
+                        scope="col"
+                        onClick={() => {
+                          requestSort('regionId');
+                        }}
+                        className={getClassNamesFor('regionId')}
```

Comment:
With VoiceOver, it's not obvious that clicking these headers will result in the table sorting.

Reply:
I changed the table headers, so it should be obvious now.